cli/command/system/info.go

```diff
@@ -254,9 +255,6 @@ func prettyPrintServerInfo(dockerCli command.Cli, info types.Info) []error {
 					for _, o := range so.Options {
 						switch o.Key {
 						case "profile":
-							if o.Value != "default" {
```

Comment:
@cpuguy83 updated; PTAL

Alternatively, I could do:

```suggestion
							if versions.LessThan(dockerCli.Client().ClientVersion(), "1.42") && o.Value != "default" && o.Value != "builtin" {
```

Which would bring a smaller patch, but I tried to also group the warning with the other warnings (printed at the end) instead of having it "half-way" the `docker info` output here. Happy to change though if you think that's an issue.